Jenkinsfile

```diff
@@ -41,12 +41,25 @@ pipeline {
 
     stages {
 
+        stage('Install the Libraries') {
+            steps {
+                withEnv(["HOME=${env.WORKSPACE}"]) {
+                    sh """
+                        source ${env.SAL_SETUP_FILE}
+
+                        cd ${env.SAL_USERS_HOME} && { curl -O ${env.PLANTUML_URL} ; cd -; }
+                        pip install sphinxcontrib-plantuml
+                    """
+                }
+            }
+        }
+
         stage('Cloning Repos') {
             steps {
                 withEnv(["HOME=${env.WORKSPACE}"]) {
                     sh """
                         git clone -b master https://github.com/lsst-dm/phosim_utils.git
-                        git clone -b ${BRANCH} https://github.com/lsst-ts/ts_wep.git
+                        git clone -b tickets/DM-27899 https://github.com/lsst-ts/ts_wep.git
```

Comment:
This can be better done by updating getBranchName:

```diff
diff --git a/Jenkinsfile b/Jenkinsfile
index 667f049..0a45ba9 100644
--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -59,7 +59,7 @@ pipeline {
                 withEnv(["HOME=${env.WORKSPACE}"]) {
                     sh """
                         git clone -b master https://github.com/lsst-dm/phosim_utils.git
-                        git clone -b tickets/DM-27899 https://github.com/lsst-ts/ts_wep.git
+                        git clone -b ${BRANCH} https://github.com/lsst-ts/ts_wep.git
                         git clone -b ${BRANCH} https://github.com/lsst-ts/ts_ofc.git
                     """
                 }
@@ -180,6 +180,7 @@ def getBranchName(changeTarget, branchName) {
     switch (branch) {
         case "master":
         case "develop":
+        case "tickets/DM-27899":
             return branch
     }
     print("!!! Returning default for branch " + branch + " !!!\n")
```

also putting this one step closer to Jenkins native:

```diff
diff --git a/Jenkinsfile b/Jenkinsfile
index 667f049..7b3e617 100644
--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -57,11 +57,14 @@ pipeline {
         stage('Cloning Repos') {
             steps {
                 withEnv(["HOME=${env.WORKSPACE}"]) {
-                    sh """
-                        git clone -b master https://github.com/lsst-dm/phosim_utils.git
-                        git clone -b tickets/DM-27899 https://github.com/lsst-ts/ts_wep.git
-                        git clone -b ${BRANCH} https://github.com/lsst-ts/ts_ofc.git
-                    """
+                    git branch: master,
+                        url: https://github.com/lsst-dm/phosim_utils.git
+
+                    git branch: ${BRANCH},
+                        url:  https://github.com/lsst-ts/ts_wep.git
+
+                    git branch: ${BRANCH},
+                        url:  https://github.com/lsst-ts/ts_ofc.git
                 }
             }
         }
@@ -180,6 +183,7 @@ def getBranchName(changeTarget, branchName) {
     switch (branch) {
         case "master":
         case "develop":
+        case "tickets/DM-27899":
             return branch
     }
     print("!!! Returning default for branch " + branch + " !!!\n")
```

Reply:
Please see the comment of this PR. Thanks!